src/main/java/org/jabref/gui/preview/PreviewViewer.java

```diff
@@ -144,7 +144,7 @@ private void highlightSearchPattern() {
                     "var markInstance = new Mark(document.getElementById(\"content\"));" +
                             "markInstance.unmark({" +
                             "  done: function(){" +
-                            "    markInstance.markRegExp(/" + pattern + "/gmi);" +
+                            "    markInstance.markRegExp(/" + pattern.replaceAll("(?<!\\\\)/", "\\\\/") + "/gmi);" +
```

Comment:
Would it be possible to extract this as constant (and thus documenting why this replacement is done)?

Reply:
Sure. I just need to get JDK 15 going for those text blocks X)
I keep getting some errors with my setup.

Reply:
@k3KAW8Pnf7mkmdSMPHz27 Source level is still jdk14, the textblock function is still preview. We only use jdk15  for compiling the jpackage stuff. (except for windows, where jdk14 is still required due to a jdk jpackage bug)

Reply:
That explains 😁